ktor-server/ktor-server-core/jvm/src/io/ktor/routing/RouteSelector.kt

```diff
@@ -32,19 +32,29 @@ public data class RouteSelectorEvaluation(
 
         /**
          * Quality of [RouteSelectorEvaluation] when a query parameter has matched
-         */
-        internal const val qualityQueryParameter = 1.0
+        */
+        public const val qualityQueryParameter: Double = 1.0
 
         /**
          * Quality of [RouteSelectorEvaluation] when a parameter with prefix or suffix has matched
          */
-        internal const val qualityParameterWithPrefixOrSuffix = 0.9
+        public const val qualityParameterWithPrefixOrSuffix: Double = 0.9
```

Comment:
The reason why ot was internal is that i was unsure about the exact value. Since it's a const val, we can't change the value in spite of that it's not a part of abi because a const val get inlined into user's code forever. Making it internal, my intention was to let us change the value if needed at any time. 

So do we have any reason/demand to access the constant outside of the module?

Reply:
Sorry, this branch was outdated and needed a rebase. This change is obsolete now.